flatliners/baseflatliner.py

```diff
@@ -38,3 +38,7 @@ def metric_name(self, x):
 
     def cluster_id(self, x):
         return self.metric_label(x, '_id')
+
+    def get_version(self, x):
```

Comment:
use the naming scheme as above.
rename to `def cluster_version`

Reply:
please rename to `cluster_version`

Reply:
Done!